lib/weka/core/instances.rb

```diff
@@ -49,12 +49,16 @@ def instances
         enumerate_instances.to_a
       end
 
-      def attributes
-        enumerate_attributes.to_a
+      def attributes(include_class_attribute = false)
```

Comment:
I would rather make it a keyword argument (`def attributes(include_class_attribute: false)`) in order to force clarity for the user. 

I don't really like to provide `attributes(true)`, because a user doesn't know what the `true` is going to do. With a keyword argument, you will have to name it explicitly: `attributes(include_class_attribute: true)`. 
A bit longer, I must admit, but way more understandable.

The same applies for `attribute_names()` below.